### DISCUSSION TITLE
MAGIX Samplitude/Sequoia Not detecting AG plugin

### DISCUSSION THREAD

**Discussion by abysalvinthomas** [apohl79/audiogridder#333] [Q&A] [answered]
MAGIX samplitude/Sequoia is not detecting audiogridder plugin...
what to do?

**Comment by abysalvinthomas** [comment, marked as answer]
Hai all 
i find out a solution by the support from magix.
scanned my vst plugin with administrator rights....done ..it worked..!!

**Reply by apohl79** [reply]
thx!
